interpreter/nodev8/v8.go

```diff
@@ -1877,6 +1928,25 @@ func (d *v8Data) readIntrospectionData(ef *pfelf.File, syms libpf.SymbolFinder)
 	}
 
 	// Add some defaults when needed
+	if d.version >= v8Ver(11, 9, 0) {
+		// the class hierarchy changed: HeapObject no longer
+		// derives from Object. This confuses gen-postmortem-metadata.py
+		// and it no longer emits class hierarchy information correctly.
+		// But ScopeInfo indeed still derives from HeapObject, so just set
+		// that manually here.
+		vms.ScopeInfo.HeapObject = true
+	}
+	if d.version >= v8Ver(12, 5, 0) && !vms.DeoptimizationData.FixedArray && !vms.DeoptimizationData.TrustedFixedArray {
```

Comment:
Can you also update the list of supported versions near the start of the file?

Reply:
Done